Makefile

```
@@ -8,7 +8,7 @@
 #   $HEADER$
 #
 
-CFLAGS ?= -g -lpthread
+CFLAGS ?= -g  -lpthread -DMULTI_MESSAGES
```

@dkuzmin72 I think that multi-message should be the main and only mode for this benchmark.
Single-message was just an intermediate step.
so let's make it a default, so no need for the MULTI_MESSAGES macro
```suggestion
CFLAGS ?= -g  -lpthread
```

---

Yep, that's OK!